test/e2e_tests/pages/conversationList.page.ts

```diff
@@ -0,0 +1,66 @@
+/*
+ * Wire
+ * Copyright (C) 2025 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+import {Locator, Page} from '@playwright/test';
+
+import {escapeHtml} from '../utils/userDataProcessor';
+
+export class ConversationListPage {
+  readonly page: Page;
+
+  readonly blockConversationMenuButton: Locator;
+
+  constructor(page: Page) {
+    this.page = page;
+
+    this.blockConversationMenuButton = page.locator('#btn-block[data-uie-name="conversation-list-options-menu"]');
```

Comment:
Does the selector need to include the class (#btn-block)? That adds a bit of fragility to the locator which the data element is supposed to resolve.